src/NetAppFiles/NetAppFiles/Volume/NewNetAppFilesVolume.cs

```diff
@@ -115,6 +116,12 @@ public class NewAzureRmNetAppFilesVolume : AzureNetAppFilesCmdletBase
         [ValidateNotNullOrEmpty]
         public PSNetAppFilesVolumeExportPolicy ExportPolicy { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable array which represents the protocol types")]
+        [ValidateNotNullOrEmpty]
+        public List<string> ProtocolType { get; set; }
```

Comment:
`ProtocolType` should have type `string[]` instead of a numerable type. See details in our [cmdlet design guideline](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/parameter-best-practices.md#array-vs-enumerable-types)

Reply:
Where do user usually get value for `ProtocolType`? We could probably use an argument completer to improve usability

Reply:
ok, updated and added argument completer.